server/routes/profile.js

```diff
@@ -1,9 +1,17 @@
-const  express= require('express');
+const express = require("express");
 const router = express.Router();
-const {multerUploads} =require('../middleware/multer')
-const { createProfile } = require("../controllers/profile");
```

Comment:
Changing the name of these should be discussed before being pushed as other code relies upon them that is in production currently.

Reply:
I think what happened is I deleted the files in that repo before you pulled it, so github decided to delete it! I guess that's why Raj recommended we not do that in the first week, it's not often an issue but in this case it was! 🤣  Great knowledge to have!